### ISSUE TITLE
GroupUrlRule unexpected behavior

### ISSUE THREAD

**Issue by akireikin** [yiisoft/yii2#7996]
This does not work:

```php
$app->get('urlManager')->rules[] = new GroupUrlRule([
    'prefix' => 'file',
    'rules' => [
        'POST document' => 'document/create',
    ],
]);
```

This works:

```php
$app->get('urlManager')->rules[] = new GroupUrlRule([
    'prefix' => 'file',
    'rules' => [
        [
            'pattern' => 'document',
            'verb' => 'POST',
            'route' => 'document/create',
        ],
    ],
]);
```

**Comment by samdark** [comment]
By doesn't work you mean 404 when parsing it or inability to create URL?

**Comment by mdmunir** [comment]
@samdark 
verb pattern not work for `yii\web\GroupUrlRule` (like at `yii\rest\UrlRule`).

**Comment by schojniak** [comment]
I fixed this in #9473 pull request.
